packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -0,0 +1,344 @@
+
+    if (result.frameWidth && result.frameHeight) {
+      this.statsResults.resolutions[mediaType][sendrecvType].width = result.frameWidth;
+      this.statsResults.resolutions[mediaType][sendrecvType].height = result.frameHeight;
+      this.statsResults.resolutions[mediaType][sendrecvType].framesSent = result.framesSent;
+      this.statsResults.resolutions[mediaType][sendrecvType].hugeFramesSent = result.hugeFramesSent;
+    }
+
+    if (result.framesReceived && result.framesDecoded && result.framesDropped) {
+      this.statsResults.resolutions[mediaType][sendrecvType].framesReceived = result.framesReceived;
+      this.statsResults.resolutions[mediaType][sendrecvType].framesDecoded = result.framesDecoded;
+      this.statsResults.resolutions[mediaType][sendrecvType].framesDropped = result.framesDropped;
+    }
+
+    if (result.trackIdentifier && mediaType !== 'audio') {
+      this.statsResults.resolutions[mediaType][sendrecvType].trackIdentifier = result.trackIdentifier;
+    }
+  };
+
+  compareSentAndReceived = (result, type) => {
+    if (!result) {
+      return;
+    }
+    if (!['remote-inbound-rtp', 'remote-outbound-rtp'].includes(result.type)) { return; }
+    const mediaType = type;
+
+    if (!mediaType) {
+      return;
+    }
+
+    if (!this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver) { this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost; }
+
+    const currentPacketLoss = result.packetsLost - this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver;
+
+    this.statsResults.internal[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost;
+    this.statsResults[mediaType].send.packetsLostOnReceiver = currentPacketLoss;
+    this.statsResults[mediaType].send.totalPacketsLostOnReceiver = result.packetsLost;
+    this.statsResults[mediaType].send.jitter = result.jitter;
+    this.statsResults[mediaType].send.roundTripTime = result.roundTripTime;
+    this.packetLossArraySender[mediaType].push(currentPacketLoss);
+
+    // Total packloss ratio on this video section of the call
+    this.statsResults[mediaType].send.overAllPacketLossRatio = this.statsResults[mediaType].send.totalPacketsLostOnReceiver > 0 ? this.statsResults[mediaType].send.totalPacketsLostOnReceiver * 100 / this.statsResults[mediaType].send.totalPacketsSent : 0;
+
+    this.statsResults[mediaType].send.currentPacketLossRatio = this.statsResults[mediaType].send.packetsLostOnReceiver > 0 ? this.statsResults[mediaType].send.packetsLostOnReceiver * 100 / this.statsResults[mediaType].send.packetsSent : 0;
+
+    if (this.statsResults[mediaType].send.maxPacketLossRatio < this.statsResults[mediaType].send.currentPacketLossRatio) {
+      this.statsResults[mediaType].send.maxPacketLossRatio = this.statsResults[mediaType].send.currentPacketLossRatio;
+    }
+
+
+    if (this.statsResults[mediaType].send.currentPacketLossRatio > 9 && mediaType === 'video') {
```

Comment:
Do we want this threshold to be configurable? Or is this a standard definition?

Reply:
Agreed, I would like it to at a minimum be defined in the constants. Better would be from config.